src/assets/locales/resources_en_US.json

```diff
@@ -11,6 +11,14 @@
     "_description": "Placeholder for country search autocomplete",
     "message": "Search for a country"
   },
+  "skyux_lookup_search_button": {
```

Comment:
I know this is just semantics but this seems odd that we are calling this resource key `button` when it is just on an icon.

Reply:
Agreed. Changed.